### PR TITLE
Update ondat cli docs

### PR DESCRIPTION
**Pull request by hubvu**
- This PR focuses on updating the Ondat CLI page and provides clear guidance on how to download and install the CLI on supported platforms (linux, windows, macOS and kubernetes distributions).
  - I have tested each deployment method (kubernetes deployment and supported workstation distributions).
  - I have removed the "Run CLI as a container" section in favour of the CLI kubernetes deployment method to keep everything as simple as possible.
- I have also removed the Ondat CLI sub command pages to reduce the number of documents to maintain long-term basis. 
  - As the product and the CLI capabilities continue evolve, a recommendation would be to refer end users to use the man pages in the CLI `storageos help` as the single source of truth for commands, subcommands and flags introduced.

### CODE REVIEW

**Review by cannischan — DISMISSED** [review]

**Comment by cannischan** [comment]
We should set up redirects for all the files that are removed

File: docs/reference/cli.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+title: "Ondat Command Line Interface (CLI) Utility"
+linkTitle: "Ondat Command Line Interface (CLI) Utility"
+weight: 1
+---
+
+## Overview
+
+- The Ondat CLI is a utility tool that is used to manage and configure Ondat resources and conduct Day-2 storage operations. The Ondat CLI is also useful for providing useful information on the state of an Ondat cluster and troubleshooting issues.
+  - The project repository is open source and can be located on  [GitHub](https://github.com/storageos/go-cli).
```
[the remark this answers is not in the document]
Comment (hubvu) [reply docs/reference/cli.md]: - fixed!

File: docs/reference/cli.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+---
```
Comment (cannischan) [inline docs/reference/cli.md:1]: @hubvu should we move this content into the _index..md file instead? The link would be https://docs.ondat.io/docs/reference/cli/ as well

Comment (hubvu) [reply docs/reference/cli.md:1]: hey @cannischan - looks like that won't be possible due to an existing `_index.md` file in the reference directory which lists the rest of the reference pages.

```yaml
.
├── _index.md
├── api.md
├── bundles
│   ├── _index.md
│   ├── diagnostic-bundle.md
│   └── support_bundle.md
├── cli.md
├── cluster-operator
│   ├── _index.md
│   ├── configuration.md
│   ├── examples.md
│   ├── install.md
│   └── upgrade.md
├── clusters.md
├── contributing.md
├── filesystems.md
├── gui.md
├── init-container.md
├── kubectl-plugin.md
├── kubernetes-object-sync.md
├── namespaces.md
├── open_source_attribution.md
├── policies.md
├── resource_requests_and_limits.md
├── scheduler
│   ├── _index.md
│   └── admission-controller.md
├── telemetry.md
└── tolerations.md

3 directories, 26 files
```

**Comment by hubvu** [comment]
@cannischan I have made the final changes to make the instructions a bit more clearer and consistent.

**Comment by cannischan** [comment]
@hubvu if we won't have anything underneath the folder `cli` then we can keep what we have right now, didn't notice that we have removed the folder altogether!

**Review by cannischan — APPROVED** [review]

**Review by cannischan — APPROVED** [review]

**Pull request merged**